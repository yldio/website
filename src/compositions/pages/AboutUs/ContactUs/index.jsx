import React from 'react';
import { Grid, Row, Col } from 'react-styled-flexboxgrid';

import PageSection from 'components/PageSection';
import { H3, H4 } from 'components/Typography';

export default () => (
  <PageSection data-selector="services:help">
    <Grid>
      <Row>
        <Col xs={12}>
          <H3 decorated>Contact Us</H3>
        </Col>
      </Row>
      <H4>UK</H4>
      <H4>Portugal</H4>
    </Grid>
  </PageSection>
);
