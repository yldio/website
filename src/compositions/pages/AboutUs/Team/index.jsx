import React, { Fragment } from 'react';
import { Grid, Row, Col } from 'react-styled-flexboxgrid';
import Flex, { FlexItem } from 'styled-flex-component';

import PageSection from 'components/PageSection';
import Ul, { Item as Li } from 'components/BulletList';
import { H3, H5, Copy } from 'components/Typography';

export default () => (
  <Fragment>
    <PageSection dark data-selector="services:help">
      <Grid>
        <Row>
          <Col xs={12} sm={5}>
            <H3 lighter decorated>
              Our Team
            </H3>
          </Col>
          <Col xs={12} sm={7}>
            <H5>Our mindset</H5>
            <Flex justifyBetween>
              <FlexItem>
                <Ul>
                  <Li light>Think differently</Li>
                  <Li light>Clients win</Li>
                  <Li light>Silicon Valley mindset</Li>
                  <Li light>Begin again, never give up</Li>
                  <Li light>Strong convictions, weakly held</Li>
                </Ul>
              </FlexItem>
              <FlexItem>
                <Ul>
                  <Li light>Everyone sweeps the floor</Li>
                  <Li light>Good enough to be dangerous</Li>
                  <Li light>Ask for forgiveness, not permission</Li>
                  <Li light>Pockets of talent</Li>
                  <Li light>Create future growth</Li>
                </Ul>
              </FlexItem>
            </Flex>
          </Col>
        </Row>
      </Grid>
    </PageSection>
    <PageSection data-selector="services:help">
      <Grid>
        <Row>
          <Col xs={12} sm={6}>
            <H3 decorated>Our Exec. Team</H3>
          </Col>
          <Col xs={12} sm={6}>
            <Copy>
              We are people who like to work out new ways of doing things
              together. We value teamwork over the individual, and our
              philosophy is to think differently, work lean with a silicon
              valley mindset and iteratively explore how things could work.
            </Copy>
          </Col>
        </Row>
        <Row>
          <Col xs={12} sm={4}>
            <H5>Nuno Job</H5>
            <Copy>Founder + Director</Copy>
            <Copy>
              Nuno Job (YLD, MCS) is the CEO of YLD, our top tier technology
              consultancy that helps top CIOs/CTOs to respond to the Innovator's
              Dilemma. Previously he was Chief Commercial at Nodejitsu where he
              was responsible for the world's largest Node.js cloud and provided
              extensive contributions to the success of Node.js as an
              enterprise-ready technology. Nuno's formative work years were
              spent in the U.S. at IBM Research and MarkLogic. He is a proud
              Sequoia alumni and a big advocate and enabler of open-source
              software.
            </Copy>
          </Col>
          <Col xs={12} sm={4}>
            <H5>Anthony Mann</H5>
            <Copy>Founder + Director</Copy>
            <Copy>
              Anthony is Design Partner and Co-Founder of YLD. He’s previously
              founded Make Us Proud - a design thinking product consultancy that
              enables companies to innovate rapidly, deliver disruptive products
              and drive exceptional customer experiences. Make Us Proud joined
              forces with YLD in 2017. Prior to this, Anthony worked at Apple,
              enabling their next generation of web products and web marketing,
              and established many of the practises we now know as Design
              Systems. In his spare time he is a mentor for startups at
              Techstars, gives talks on product and design, and has even curated
              an exhibition with the Tate.
            </Copy>
          </Col>
          <Col xs={12} sm={4}>
            <H5>Fábio Oliveira</H5>
            <Copy>CEO of AddCode</Copy>
            <Copy>
              Learning addict, pathologically curious. Photographs stuff. Star
              Wars connoisseur. Has a soft spot for cats. Computer Science
              Graduate Software Engineer (9+ years) Pragmatic Analyst and
              Problem Solver (29+ years) Focused on Portuguese companies
              operations and finance (2+ years) Currently managing operations of
              YLD Portugal companies. Specialties: Software Architecture, Bug
              Hunting
            </Copy>
          </Col>
          <Col xs={12} sm={4}>
            <H5>Hema Patel</H5>
            <Copy>COO</Copy>
            <Copy>
              Hema serves as COO of YLD and provides strategic oversight of all
              operational functions and leads company growth. She is a senior
              executive who brings 25 years of experience in both strategy and
              global director positions at Adobe, Facebook, Microsoft and spent
              12 years in publishing, leading commercial teams at News Corp and
              the FT. Henley business school trained with a degree in
              Accountancy. Hema is a mentor at the Microsoft Accelerator and a
              passionate sailor.
            </Copy>
          </Col>
          <Col xs={12} sm={4}>
            <H5>Bradley Stott</H5>
            <Copy>CCO</Copy>
            <Copy>XXX</Copy>
          </Col>
          <Col xs={12} sm={4}>
            <H5>Tracey Bianchi-Maisuria</H5>
            <Copy>CFO</Copy>
            <Copy>
              Tracey is our CFO, and Stanford educated. She is responsible for
              all financial internal controls and provides strategic
              recommendations to leadership based on financial analysis and
              projections, and revenue/expense analysis. Previously Tracey
              worked at both the Royal Society and the British Safety Council.
            </Copy>
          </Col>
          <Col xs={12} sm={4}>
            <H5>Carlos Vilhena</H5>
            <Copy>VP Engineering</Copy>
            <Copy>
              Carlos is the VP of Engineering at YLD and leads our engineering
              team, comprising of 90 engineers across two markets. He oversees
              all technical recruitment, the engineering communities, developer
              advocacy, career development, our open source strategy, in
              addition to the technical roadmap and training. Carlos has
              experience managing engineering teams across EMEA markets. Prior
              to YLD, Carlos was a startup founder and an alumni of Forward
              Partners, a leading UK investment firm. He has also served as a
              tech lead at GDS, a Government organisation focusing on digital
              transformation and is educated to MSc level in Computer Science,
              specialising in Formal Methods.
            </Copy>
          </Col>
          <Col xs={12} sm={4}>
            <H5>Gareth xxxx</H5>
            <Copy>Associate Partner</Copy>
            <Copy>
              Gareth is YLD’s first Associate Partner and leads our Principals.
              He brings 20 years’ experience in the delivery of technology
              strategies and solutions for high profile international clients
              including Nike and Heineken. He most recently led growth of the
              Interactive Development teams at IBM iX. Prior to this, he was
              Group Technical director at AKQA London, for many years through to
              their exit. Gareth also brings consulting experience from Cap
              Gemini and software engineering expertise.
            </Copy>
          </Col>
          <Col xs={12} sm={4}>
            <H5>Danielle Leach</H5>
            <Copy>CMO</Copy>
            <Copy>
              Danielle is the CMO of YLD. She is responsible for leading the
              marketing team across marketing strategy, PR & communications and
              customer-facing events support. Danielle brings over 25 vears of
              international experience as a PR & communications strategist. She
              has held positions at Google and Ericsson and has consulted to
              Microsoft, eBay and SAP. Danielle holds an M.A. in Arts Criticism
              and Policy Management and a B.A. in English Literature and History
              of Art. She holds the CAM Foundation Communication Certificate and
              attended Ashridge Centre for Business and Society gaining
              certification in Managing Corporate Community Investment.
            </Copy>
          </Col>
        </Row>
      </Grid>
    </PageSection>
  </Fragment>
);

// <PageSection data-selector="services:help">
//   <Grid>
//     <Cards justifyBetween>
//       <FlexItem>
//         <Card
//           title="Private Training"
//           points={[
//             'Hands-on learning experience, in-person classes',
//             'Tailoried training program delivered on-site to suit your needs',
//             'Ensures your software engineers share a baseline'
//           ]}
//         />
//       </FlexItem>
//       <FlexItem>
//         <Card
//           title="Pairing"
//           points={[
//             'Ongoing tutoring that covers all aspect of tech and product delivery',
//             'Seasoned YLD engineers and designers available to answer questions at all times',
//             'Experienced supervisio for code review'
//           ]}
//         />
//       </FlexItem>
//       <FlexItem>
//         <Card
//           title="Certification"
//           points={[
//             'Gain recognition for expertise and ability to employ best practices',
//             'Assess engineering candidates expertise and compare the results with your team’s performance'
//           ]}
//         />
//       </FlexItem>
//     </Cards>
//   </Grid>
//   <Grid>
//     <Row>
//       <Col xs={12} sm={6}>
//         <H2 decorated>
//           Our goal is never to just deliver but always to also upskill your
//           whole team.
//         </H2>
//       </Col>
//       <Col xs={12} sm={6}>
//         <Copy>
//           We are an agile, flexible and passionate team consisting of strategy,
//           design and engineering.
//           <br />
//           <br />
//           We aspire to bring you the most up-to-date technology, with the
//           minimal hassle possible.
//           <br />
//           <br />
//           We take a hands on approach in our work, engaging directly with you in
//           every stage of the development process.
//           <br />
//           <br />
//           We take ownership of the delivery of the project or we can integrate
//           at critical points with your team according to your needs.
//           <br />
//           <br />
//           We enable your organisation to deliver world-class technology and user
//           experiences, long after we leave.
//         </Copy>
//       </Col>
//     </Row>
//   </Grid>
// </PageSection>;
