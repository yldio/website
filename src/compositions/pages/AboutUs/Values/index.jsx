import React, { Fragment } from 'react';
import PropTypes from 'prop-types';
import TileGrid, { Tile } from 'components/TileGrid';
import PageSection from 'components/PageSection';
import { Copy, H2 } from 'components/Typography';
import {
  TileContent,
  TileContentTitle,
  TileContentBody,
  Header,
  Description,
  OurValues,
  ValuesSection,
} from './styled';

export const Value = ({ header, children }) => (
  <ValuesSection>
    <Header>{header}</Header>
    <Description>{children}</Description>
  </ValuesSection>
);

const Values = () => (
  <Fragment>
    <PageSection data-selector="about-us:our-values">
      <H2>Our Values</H2>

      <Copy light>
        We help you future proof your business though a new style of consulting,
        ground up. We connect and educate engineering, product and leadership to
        help you navigate digital transformation, grow your business and tackle
        core customer problems. We’re one of the most renowned technology
        companies in Europe.
      </Copy>
      <Copy>
        Some amazing support copy with go here to reenforce whatever section it
        is relevant too. Some amazing support copy with go here to reenforce
        whatever section it is relevant too. Some amazing support copy with go
        here to reenforce whatever section it is relevant too. Some amazing
        support copy with go here to reenforce whatever section it is relevant
        too.
      </Copy>
    </PageSection>
    <PageSection>
      <TileGrid>
        <OurValues>
          <Value header="Your opportunity is limitless">
            You are the driver of change. We never give up, we begin again. Be
            adventuruous. Push the boundaries. Go the extra mile.
          </Value>
          <Value header="Have integrity, be honest">
            Lead by example. No “bullshit”. Be real, have respect.
          </Value>
          <Value header="Get better as a team">
            Collaborate, collaborate, collaborate. Never stop learning annd
            learn together - innovation can be the innovation you inspire in
            others. Be inquisitive. Ask why, and be preparered to be asked why
            of yourself.
          </Value>
          <Value header="Recognise success">
            Understand the purpose. When you succeed, help others do the same.
            Celebrate, even when we fail.
          </Value>
          <Value header="Have courage, stay humble">
            Put our partners success ahead of our own and treat them as we would
            our own team. Have strong opinions, weakly held. Good enough isn’t
            good enough.
          </Value>
        </OurValues>
      </TileGrid>
    </PageSection>
    <PageSection wide>
      <TileGrid>
        <Tile>
          <TileContent>
            <TileContentTitle>Services</TileContentTitle>
            <TileContentBody>
              We enable your organisation to deliver world-class technology and
              user experiences, long after we leave.
            </TileContentBody>
          </TileContent>
        </Tile>
        <Tile colour="darker">
          <TileContent>
            <TileContentTitle>Community</TileContentTitle>
            <TileContentBody>
              We define ourselves by the people who represent us.
            </TileContentBody>
          </TileContent>
        </Tile>
      </TileGrid>
    </PageSection>
  </Fragment>
);

export default Values;

Value.propTypes = {
  header: PropTypes.string.isRequired,
  children: PropTypes.string.isRequired,
};
