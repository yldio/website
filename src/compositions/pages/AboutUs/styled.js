import styled from 'styled-components';
import { spacing } from 'styles/spacing';
import Link from 'components/Link';
import remcalc from 'remcalc';
import { highlight } from 'styles/colours';

export const People = styled.ul`
  padding: 0;
  margin: 0;
  display: grid;
  grid-template-columns: repeat(auto-fill, minmax(250px, 1fr));
  grid-column-gap: ${spacing(6)};
`;

export const WorkWithUs = styled.div`
  display: flex;
  font-size: ${remcalc(36)};
  align-items: center;
  justify-content: center;
  flex-direction: column;
`;

export const Anchor = styled(Link)`
  font-size: ${remcalc(36)};
  color: ${highlight};
`;

export const LinkWrapper = styled.div`
  margin-bottom: ${spacing(7)};
  color: ${highlight};
`;
