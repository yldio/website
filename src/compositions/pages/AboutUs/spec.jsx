import React from 'react';
import { shallow } from 'enzyme';
import 'jest-styled-components';

import AboutUs from './index';

test('components/AboutUs snapshot', () => {
  const wrapper = shallow(<AboutUs />);

  expect(wrapper).toMatchSnapshot();
});
