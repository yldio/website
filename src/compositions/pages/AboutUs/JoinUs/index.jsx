import React from 'react';
import { Grid, Row, Col } from 'react-styled-flexboxgrid';
import styled from 'styled-components';

import PageSection from 'components/PageSection';
import { H4, Copy } from 'components/Typography';
import Button from 'components/Button';

const Section = styled(PageSection)`
  background-color: #5e5cb2;
`;

export default () => (
  <Section data-selector="home:community">
    <Grid>
      <Row>
        <Col xs={12}>
          <H4 lighter>Join Us</H4>
          <Copy>
            <br />
          </Copy>
        </Col>
        <Col xs={12} sm={8}>
          <Copy lighter>
            If you share a similar vision, or a passion for design, software
            engineering and open source, we would love to hear from you.
          </Copy>
        </Col>
        <Col xs={12} sm={4}>
          <Row center="xs">
            <Col xs={12}>
              <Button transparent>View openings</Button>
            </Col>
          </Row>
        </Col>
      </Row>
    </Grid>
  </Section>
);
