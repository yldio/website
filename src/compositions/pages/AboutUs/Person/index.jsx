import React from 'react';
import PropTypes from 'prop-types';
import Link from 'components/Link';
import { Copy } from 'components/Typography';

import { Img, Title } from './styled';

const Picture = ({ picture, name, title, twitter, linkedin, children }) => (
  <li>
    <Img src={picture} alt={name} width="180" height="180" />
    <Copy bold> {name}</Copy>
    <Title> {title}</Title>
    <Copy> {children}</Copy>
    <Copy>
      <Link href={twitter} target="_blank">
        Twitter
      </Link>
    </Copy>
    <Copy>
      <Link href={linkedin} target="_blank">
        LinkedIn
      </Link>
    </Copy>
  </li>
);

export default Picture;

Picture.propTypes = {
  children: PropTypes.string.isRequired,
  name: PropTypes.string.isRequired,
  picture: PropTypes.string.isRequired,
  title: PropTypes.string.isRequired,
  twitter: PropTypes.string.isRequired,
  linkedin: PropTypes.string.isRequired,
};
