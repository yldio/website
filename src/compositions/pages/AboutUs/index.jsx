import React, { Fragment } from 'react';
import { Grid, Row } from 'react-styled-flexboxgrid';

import { H2, Copy } from 'components/Typography';
import PageHero from 'components/PageHero';
import Container from 'components/Container';

import { Section } from './styled';
import Stats from './Stats';
import Team from './Team';
import ContactUs from './ContactUs';
import JoinUs from './JoinUs';

export default () => (
  <Fragment>
    <Section data-selector="services:hero">
      <Container>
        <Grid>
          <Row>
            <PageHero title={<H2 lighter>About Us</H2>} white big>
              <Copy lighter>
                Curious thinkers, technology pioneers and custom software
                engineers. Our teams are people who want to use technology to
                build, to create digital transformation. We innovate by doing,
                working lean and sharing knowledge so everyone grows.
              </Copy>
            </PageHero>
          </Row>
        </Grid>
      </Container>
    </Section>
    <Stats />
    <Team />
    <ContactUs />
    <JoinUs />
  </Fragment>
);

// import React, { Fragment } from 'react';
// import PageHero from 'components/PageHero';
// import PageSection from 'components/PageSection';
// import { Copy, H2 } from 'components/Typography';
// import Person from './Person';
// import { People, WorkWithUs, Anchor, LinkWrapper } from './styled';
// import Values from './Values';

// import nuno from './Person/nuno.png';

// export default () => (
//   <Fragment>
//     <PageSection data-selector="about-us:hero">
//       <PageHero
//         before="about us"
//         title="Being a great technology focused company is hard. We get it."
//       >
//         Adopting a rapid, quality focused approach to building and maintaining
//         your core technology, scaling a world-class user-experience and brand,
//         and releasing value to your customers and your employees at a fast pace
//         is a challenge, and this is only getting faster.
//       </PageHero>
//     </PageSection>
//     <PageSection data-selector="about-us:our-team">
//       <H2>Our Team</H2>

//       <Copy>
//         We have team members from a broad spectrum of backgrounds and incredible
//         technology companies.
//       </Copy>
//     </PageSection>
//     <PageSection data-selector="about-us:team">
//       <People>
//         <Person
//           picture={nuno}
//           name="Nuno Job"
//           title="Founder + Director"
//           twitter="https://twitter.com/dscape"
//           linkedin="https://twitter.com/dscape"
//         >
//           Nuno is the founder and director at YLD. The previous CCO at
//           Nodejitsu, Nuno was responsible for business development and sales in
//           the world’s largest Node.js cloud. Now, he leads the YLD team and
//           inspires all the creative minds that make up the company.
//         </Person>
//         <Person
//           picture={nuno}
//           name="Nuno Job"
//           title="Founder + Director"
//           twitter="https://twitter.com/dscape"
//           linkedin="https://twitter.com/dscape"
//         >
//           Nuno is the founder and director at YLD. The previous CCO at
//           Nodejitsu, Nuno was responsible for business development and sales in
//           the world’s largest Node.js cloud. Now, he leads the YLD team and
//           inspires all the creative minds that make up the company.
//         </Person>
//         <Person
//           picture={nuno}
//           name="Nuno Job"
//           title="Founder + Director"
//           twitter="https://twitter.com/dscape"
//           linkedin="https://twitter.com/dscape"
//         >
//           Nuno is the founder and director at YLD. The previous CCO at
//           Nodejitsu, Nuno was responsible for business development and sales in
//           the world’s largest Node.js cloud. Now, he leads the YLD team and
//           inspires all the creative minds that make up the company.
//         </Person>
//       </People>
//     </PageSection>
//     <Values />
//     <WorkWithUs data-selector="about-us:work-width-us">
//       <H2>Work with us</H2>
//       <LinkWrapper>
//         <Anchor href="mailto:hello@yld.io">hello@yld.io</Anchor> /{' '}
//         <Anchor href="tel:4402035144678">+44 (0) 203 514 4678</Anchor>
//       </LinkWrapper>
//     </WorkWithUs>
//   </Fragment>
// );
