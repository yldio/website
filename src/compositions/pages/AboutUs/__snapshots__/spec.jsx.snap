// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/AboutUs snapshot 1`] = `
<React.Fragment>
  <PageSection
    data-selector="about-us:hero"
    gradient={false}
    gradientHeight="50%"
  >
    <PageHero
      before="about us"
      title="Being a great technology focused company is hard. We get it."
    >
      Adopting a rapid, quality focused approach to building and maintaining your core technology, scaling a world-class user-experience and brand, and releasing value to your customers and your employees at a fast pace is a challenge, and this is only getting faster.
    </PageHero>
  </PageSection>
  <PageSection
    data-selector="about-us:our-team"
    gradient={false}
    gradientHeight="50%"
  >
    <Typography__H2>
      Our Team
    </Typography__H2>
    <Typography__base>
      We have team members from a broad spectrum of backgrounds and incredible technology companies.
    </Typography__base>
  </PageSection>
  <PageSection
    data-selector="about-us:team"
    gradient={false}
    gradientHeight="50%"
  >
    <styled__People>
      <Picture
        linkedin="https://twitter.com/dscape"
        name="Nuno Job"
        picture="/src/compositions/pages/AboutUs/Person/nuno.png"
        title="Founder + Director"
        twitter="https://twitter.com/dscape"
      >
        Nuno is the founder and director at YLD. The previous CCO at Nodejitsu, Nuno was responsible for business development and sales in the world’s largest Node.js cloud. Now, he leads the YLD team and inspires all the creative minds that make up the company.
      </Picture>
      <Picture
        linkedin="https://twitter.com/dscape"
        name="Nuno Job"
        picture="/src/compositions/pages/AboutUs/Person/nuno.png"
        title="Founder + Director"
        twitter="https://twitter.com/dscape"
      >
        Nuno is the founder and director at YLD. The previous CCO at Nodejitsu, Nuno was responsible for business development and sales in the world’s largest Node.js cloud. Now, he leads the YLD team and inspires all the creative minds that make up the company.
      </Picture>
      <Picture
        linkedin="https://twitter.com/dscape"
        name="Nuno Job"
        picture="/src/compositions/pages/AboutUs/Person/nuno.png"
        title="Founder + Director"
        twitter="https://twitter.com/dscape"
      >
        Nuno is the founder and director at YLD. The previous CCO at Nodejitsu, Nuno was responsible for business development and sales in the world’s largest Node.js cloud. Now, he leads the YLD team and inspires all the creative minds that make up the company.
      </Picture>
    </styled__People>
  </PageSection>
  <Values />
  <styled__WorkWithUs
    data-selector="about-us:work-width-us"
  >
    <Typography__H2>
      Work with us
    </Typography__H2>
    <styled__LinkWrapper>
      <styled__Anchor
        href="mailto:hello@yld.io"
      >
        hello@yld.io
      </styled__Anchor>
       /
       
      <styled__Anchor
        href="tel:4402035144678"
      >
        +44 (0) 203 514 4678
      </styled__Anchor>
    </styled__LinkWrapper>
  </styled__WorkWithUs>
</React.Fragment>
`;
